.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,115 @@
+name: CI Release Images
+
+on:
+  workflow_call:
+    inputs:
+      repository:
+        description: 'Name of the repo to download server artifact from'
+        type: string
+      run-id:
+        description: 'Id of the run to download server artifact from'
+        type: string
+      serverArtifact:
+         description: 'Artifact name to download'
+         type: string
+      tags:
+        description: 'Tags for the image'
+        type: string
+      push:
+         description: 'If true push image.'
+         default: true
+         type: boolean
+      branch:
+         description: 'infinispan-image branch to checkout.'
+         default: main
+         type: string
+    secrets:
+      token:
+         required: true
+         description: 'GH token'
+      quay_user:
+         required: true
+      quay_pass:
+         required: true
+
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'Name of the repo to download server artifact from'
+        type: string
+      run-id:
+        description: 'Id of the run to download server artifact from'
+        type: string
+      serverArtifact:
```

Comment:
I don't mind whether we use camelCase or kebab-case, but let's try to be consistent with the input element across our various workflows.

Reply:
let's go with camel